package.json

```diff
@@ -38,7 +38,7 @@
     "focus-trap-react": "^10.2.3",
     "lodash.debounce": "^4.0.8",
     "lucide-react": "^0.341.0",
-    "next": "14.0.1",
+    "next": "^14.1.4",
```

Comment:
I was getting this error when I used latest. https://makeswifthq.slack.com/archives/C014SCS4G75/p1711048297151889

Please make sure you can open the preview URL in the builder before you ship this. Hopefully it works.

Reply:
Looks good to me